sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/scripting/UnifiedExpressionCheck.java

```diff
@@ -122,6 +122,13 @@ public void startDocument(List<Node> nodes) {
   public void startElement(TagNode element) {
 
     for (Attribute attribute : element.getAttributes()) {
+
+      // Ignore thymeleaf expressions
+      String name = attribute.getName();
+      if (name.startsWith("th:")) {
+        continue;
+      }
```

Comment:
Maybe I should ignore thymeleaf expressions more globally. For instance, in the definition of :
 ``` java 
isUnifiedExpression(value) //line 135
```
But I was not sure of the side effect. Let me know if you have an opinion on this.

Reply:
I think it's OK to to ingore thmyleaf expressions globally: the 2 other places where `isUnifiedExpression` is called actually make sense to keep the current logic for thmyleaf expressions in my opinion